manifests/j/jqlang/jq/1.6/jqlang.jq.installer.yaml

```diff
@@ -1,14 +1,14 @@
-# Created using wingetcreate 1.1.2.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+# Created with YamlCreate.ps1 v2.2.9 $debug=AUSU.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
 
 PackageIdentifier: jqlang.jq
-PackageVersion: 1.6
+PackageVersion: "1.6"
+InstallerType: portable
```

Comment:
This directive is mislocated. It should go under `AppsAndFeaturesEntries`

Reply:
No, in this case it is correct. Installer type can appear at several levels and has different meaning depending on its location.

Reply:
Ah, I see. I will fix it in my PR.